Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -49,29 +61,76 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   return 0;
 }
 
+// Retrieves the fuzzing target from the FUZZING_TARGET environment variable.
+// Default target is kNone if the environment variable is empty, not set, or
+// could not be interpreted. Should be kept in sync with FuzzingTarget.
+FuzzingTarget GetFuzzingTarget() {
+  char *fuzzing_target_env = std::getenv("FUZZING_TARGET");
+
+  if (fuzzing_target_env == NULL || strlen(fuzzing_target_env) == 0) {
```

Comment:
Nit: prefer `nullptr` to `NULL` (added in C++11, `nullptr` has all the same uses as `NULL` but without some gotchas. `NULL` is some sort of an `int`, which may lead to surprising conversions in some contexts. While it doesn't apply here, `nullptr` is strictly superior, so there really is no reason to not just forget about `NULL` in the new code).

(example of a surprising conversion:
```cpp
#include <iostream>

struct Foo {};

void foo(unsigned) {
  std::cout << "unsigned\n";
}
template <typename T>
void foo(T*) {
  std::cout << "pointer\n";
}

int main() {
  foo(NULL); // Prints "unsigned"
}
```
)

Reply:
Wow! Okay, thanks for the example!